tempesta_fw/t/unit/user_space/percentiles.c

```diff
@@ -264,7 +264,7 @@ tfw_stat_adjust(TfwPcntRanges *rng, int r)
 		goto out;
 
 	printf("  -- range %d has outlier %lu (avg=%lu total=%lu) at"
-	       " bucket %lu\n", r, max, sum / cnt, cnt, i_max);
+	       " bucket %lu\n", r, max, sum / cnt, sum, i_max);
```

Comment:
Just tried to start Tempesta with Apache upstream with default CentOS configuration and got following results for one request from Chrome browser:

        [root@localhost tempesta]# cat /proc/tempesta/servers/127.0.0.1\:8080 
        Minimal response time		: 0ms
        Median  response time		: 0ms
        Maximum response time		: 0ms
        Percentiles
        50%:	3ms
        75%:	0ms
        90%:	0ms
        95%:	0ms
        [root@localhost tempesta]# cat /proc/tempesta/perfstat 
        SS pfl hits				: 6
        SS pfl misses				: 12
        Cache hits				: 0
        Cache misses				: 0
        Client messages received		: 6
        Client messages forwarded		: 6
        Client messages served from cache	: 0
        Client messages parsing errors		: 0
        Client messages filtered out		: 0
        Client messages other errors		: 0
        Client connection attempts		: 4
        Client established connections		: 4
        Client connections active		: 4
        Client RX bytes				: 2983
        Server messages received		: 6
        Server messages forwarded		: 6
        Server messages parsing errors		: 0
        Server messages filtered out		: 0
        Server messages other errors		: 0
        Server connection attempts		: 8
        Server established connections		: 8
        Server connections active		: 4
        Server RX bytes				: 13545
        [root@localhost tempesta]# cat etc/tempesta_fw.conf 
        listen 172.16.0.5:80;
        server 127.0.0.1:8080;
        
        cache 1;
        
        sticky name=__TFW__ enforce;
        sticky_secret "f00)9eR59*_/22";
        sess_lifetime 90;
        
        frang_limits {
                request_rate 20;
                request_burst 15;
                connection_rate 8;
                connection_burst 6;
                concurrent_connections 8;
                client_header_timeout 20;
                client_body_timeout 10;
                http_uri_len 1024;
                http_field_len 256;
                http_ct_required false;
                http_methods get post head;
                http_ct_vals "text/plain" "text/html";
                http_header_chunk_cnt 10;
                http_body_chunk_cnt 0;
        	http_host_required 0;
        }

I.e. 50% of requests are less than 3ms. But what about the rest of 3 requests (performance statistic shows that there are 6 requests).